qiskit/ignis/verification/randomized_benchmarking/experiments/utils.py

```diff
@@ -0,0 +1,440 @@
+        Therefore the EPC can be written as:
+
+        .. math::
+
+            EPC = 1 - (1 - EPG_{U2})^{N_{U2}} (1 - EPG_{U3})^{N_{U3}}.
+
+        Because ``u2`` and ``u3`` gates are respectively implemented by a single and two
+        half-pi pulses with virtual-Z rotations, we assume :math:`EPG_{U3} = 2EPG_{U2}`.
+        Using this relation in the limit of :math:`EPG_{U2} \ll 1`:
+
+        .. math::
+
+            EPC & = 1 - (1 - EPG_{U2})^{N_{U2}} (1 - 2 EPG_{U2})^{N_{U3}} \\
+                & \simeq EPG_{U2}(N_{U2} + 2 N_{U3}).
+
+        Finally the EPG of each basis gate can be written using EPC and number of gates:
+
+        .. math::
+
+            EPG_{U1} &= 0 \\
+            EPG_{U2} &= EPC / (N_{U2} + 2 N_{U3}) \\
+            EPG_{U3} &= 2 EPC / (N_{U2} + 2 N_{U3})
+
+        To run this function, you first need to run a standard 1Q RB experiment with transpiled
+        ``QuantumCircuit`` and count the number of basis gates composing the RB circuits.
+
+        .. jupyter-execute::
+
+            import pprint
+            import qiskit.ignis.verification.randomized_benchmarking as rb
+
+            # assuming we ran 1Q RB experiment for qubit 0
+            gpc = {0: {'cx': 0, 'u1': 0.13, 'u2': 0.31, 'u3': 0.51}}
+            epc = 1.5e-3
+
+            # calculate 1Q EPGs
+            epgs = rb.rb_utils.calculate_1q_epg(gate_per_cliff=gpc, epc_1q=epc, qubit=0)
+            pprint.pprint(epgs)
+
+        In the example, ``gpc`` can be generated by :func:`gates_per_clifford`.
+        The output of the function ``epgs`` can be used to calculate EPG of CNOT gate
+        in conjugation with 2Q RB results, see :func:`calculate_2q_epg`.
+
+        Note:
+            This function presupposes the basis gate consists
+            of ``u1``, ``u2`` and ``u3``.
+
+        Args:
+            gate_per_cliff: dictionary of gate per Clifford. see :func:`gates_per_clifford`.
+            epc_1q: EPC fit from 1Q RB experiment data.
+            qubit: index of qubit to calculate EPGs.
+
+        Returns:
+            Dictionary of EPGs of single qubit basis gates.
+
+        Raises:
+            QiskitError: when ``u2`` or ``u3`` is not found, ``cx`` gate count is nonzero,
+                or specified qubit is not included in the gate count dictionary.
+        """
+        if qubit not in gate_per_cliff:
+            raise QiskitError('Qubit %d is not included in the `gate_per_cliff`' % qubit)
+
+        gpc_per_qubit = gate_per_cliff[qubit]
+
+        if 'u3' not in gpc_per_qubit or 'u2' not in gpc_per_qubit:
+            raise QiskitError('Invalid basis set is given. Use `u1`, `u2`, `u3` for basis gates.')
+
+        n_u2 = gpc_per_qubit['u2']
+        n_u3 = gpc_per_qubit['u3']
+
+        if gpc_per_qubit.get('cx', 0) > 0:
+            raise QiskitError('Two qubit gate is included in the RB sequence.')
+
+        return {'u1': 0, 'u2': epc_1q / (n_u2 + 2 * n_u3), 'u3': 2 * epc_1q / (n_u2 + 2 * n_u3)}
```

Comment:
`u1, u2, u3` are deprecated, and should be replaced: `u1 -> rz, u2 -> sx, u3 -> x`
How to estimate the error ratio between `x` and `sx`? is it `2:1` (like `u3:u2`) or `1:1` or something else?
This should also affect the functions: `calculate_2q_epg`, `calculate_1q_epc`, `calculate_2q_epc`